pkg/rsyslog/Dockerfile

```diff
@@ -1,10 +1,12 @@
+FROM linuxkit/memlogd:v0.5-amd64 as memlogd
```

Comment:
As I pointed out before @gkodali-zededa -- we should avoid these types of dependencies between packages unless they are absolutely the only way to accomplish a given task. See my thoughts on how we can do it differently bellow.

Reply:
Also -amd64 is not correct

Reply:
removed -amd64